tools/push_api_gh_pages.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -ev
+
+GH_REPO="@github.com/elopio/snapcraft.git"
+
+FULL_REPO="https://$GH_TOKEN$GH_REPO"
+
+cd docs/api/_build/html
```

Comment:
the plan was to delete `docs` and add `snappy-docs` as a submodule, can this be somewhere else?